crates/core/src/client/config.rs

```diff
@@ -11,10 +11,21 @@ use url::Url;
 use super::constants::{KATANA_RPC_URL, MADARA_RPC_URL};
 use super::errors::ConfigError;
 
-fn get_env_var(name: &str) -> Result<String, ConfigError> {
+fn env_var(name: &str) -> Result<String, ConfigError> {
     std::env::var(name).map_err(|_| ConfigError::EnvironmentVariableMissing(name.into()))
 }
 
+fn field_element_from_env(var_name: &str) -> Result<FieldElement, ConfigError> {
+    let env_var = env_var(var_name)?;
+
+    FieldElement::from_hex_be(&env_var).map_err(|_| {
+        ConfigError::EnvironmentVariableSetWrong(format!(
+            "{} should be provided as a hex string, got {}",
```

Comment:
Isn't there an error case where the input is a valid hex string, but bigger than 31 bytes or 32 bytes i.e. longer than 65 or 66 chars?